build/avalanchego-apis/exchange-chain-x-chain-api.md

```diff
@@ -438,6 +438,84 @@ curl -X POST --data '{
 }
 ```
 
+### avm.createAsset
+
+Create a new variable-cap, fungible asset. No units of the asset exist at initialization. Minters can mint units of this asset using `createMintTx`, `signMintTx` and `issueTx`. The asset can be sent with `avm.send`
+
+#### **Signature**
+
+```cpp
+avm.createAsset({
+    name: string,
+    symbol: string,
+    denomination: int, //optional
+    minterSets: []{
+        minters: []string,
+        threshold: int
+    },
+    from: []string, //optional
+    changeAddr: string, //optional
+    username: string,
+    password: string
+}) ->
+{
+    assetID: string,
+    changeAddr: string,
+}
+```
+
+* `name` is a human-readable name for the asset. Not necessarily unique.
+* `symbol` is a shorthand symbol for the asset. Between 0 and 4 characters. Not necessarily unique. May be omitted.
+* `denomination` determines how balances of this asset are displayed by user interfaces. If denomination is 0, 100 units of this asset are displayed as 100. If denomination is 1, 100 units of this asset are displayed as 10.0. If denomination is 2, 100 units of this asset are displays as .100, etc.
+* `minterSets` is a list where each element specifies that `threshold` of the addresses in `minters` may together mint more of the asset by signing a minting transaction.
+* `from` are the addresses that you want to use for this operation. If omitted, uses any of your addresses as needed.
+* `changeAddr` is the address any change will be sent to. If omitted, change is sent to one of the addresses controlled by the user.
+* `username` pays the transaction fee.
+* `assetID` is the ID of the new asset.
+* `changeAddr` in the result is the address where any change was sent.
+
+#### **Example Call**
+
+```cpp
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     : 1,
+    "method" :"avm.createVariableCapAsset",
```

Comment:
This `avm.createVariableCapAsset` should be `avm.createAsset"`

Also, it looks like this RPC needs to be added to AvalancheJS